js/packages/components/modals/DeleteAccount.tsx

```diff
@@ -138,6 +139,8 @@ const DeleteAccountContent: React.FC<{}> = () => {
 					]}
 					value={deleteConfirmation}
 					onChangeText={setDeleteConfirmation}
+					autoCorrect={false}
+					autoCapitalize='none'
```

Comment:
if we disable `autoCapitalize` we should also set the `DELETE_STR` to 'delete' (without cap) IMO

Reply:
Hah, I agree 100%, I did it in my last push, then set it back. Doing.